src/test/java/org/broadinstitute/hellbender/tools/spark/pipelines/ReadsPipelineSparkIntegrationTest.java

```diff
@@ -57,31 +56,23 @@ private String getResourceDir(){
 
     @DataProvider(name = "ReadsPipeline")
     public Object[][] createReadsPipelineSparkTestData() {
-        final String GRCh37Ref_2021 = b37_reference_20_21;
         final String GRCh37Ref2bit_chr2021 = b37_2bit_reference_20_21;
         final String hiSeqBam_chr20 = getResourceDir() + "CEUTrio.HiSeq.WGS.b37.ch20.1m-1m1k.NA12878.noMD.noBQSR.bam";
-        final String hiSeqBam_chr20_queryNameSorted = getResourceDir() + "CEUTrio.HiSeq.WGS.b37.ch20.1m-1m1k.NA12878.noMD.noBQSR.queryNameSorted.bam";
-        final String hiSeqCram_chr20 = getResourceDir() + "CEUTrio.HiSeq.WGS.b37.ch20.1m-1m1k.NA12878.noMD.noBQSR.cram";
         final String dbSNPb37_20 = getResourceDir() + DBSNP_138_B37_CH20_1M_1M1K_VCF;
         final String more20Sites = getResourceDir() + "dbsnp_138.b37.20.10m-10m100.vcf"; //for testing 2 input files
 
-        final String expectedSingleKnownSites = "expected.CEUTrio.HiSeq.WGS.b37.ch20.1m-1m1k.NA12878.noMD.noBQSR.md.bqsr.bam";
-        final String expectedMultipleKnownSites = "expected.MultiSite.reads.pipeline.bam";
-        final String expectedMultipleKnownSitesCram = "expected.MultiSite.reads.pipeline.cram";
+        final String expectedMultipleKnownSites = "expected.MultiSite.reads.pipeline.vcf";
 
         return new Object[][]{
                 // input local, computation local.
-                //Note: these output files were created by running Picard 1.130 and GATK3.46
```

Comment:
I initially removed them due to expediency, so I've gone back and added them back. The only ones I think we don't need are the shuffle join partitioner ones - they fail due to a serialization error, but we plan on removing the shuffle join partitioner since broadcast or overlaps partitioner are superior.

Also, CRAM isn't working since it doesn't work with 2bit, and HaplotypeCaller requires 2bit. I'll add a note about this.